go/consensus/tendermint/db/badger/badger.go

```diff
@@ -9,10 +9,10 @@ import (
 	"strings"
 	"sync"
 
+	dbm "github.com/cometbft/cometbft-db"
```

Comment:
```suggestion
	cdb "github.com/cometbft/cometbft-db"
```
Have no idea where `dbm` comes from :)

Reply:
We use this elsewhere for the Tendermint DB import, so I standardized on it :)

Reply:
Your changes make sense, but not sure why was named `dbm` in the first place. As we switched to `cometbft`, we could rename to `cdb` or something similar everywhere.

Reply:
What if they rename it again?